src/Formatter/LegacyFormatter.php

```diff
@@ -79,22 +45,18 @@ public function __construct($options = [])
      * Sets the data (done by the parser)
      *
      * @param array $settings
-     *
-     * @return \BrowscapPHP\Formatter\PhpGetBrowser
      */
-    public function setData(array $settings)
+    public function setData(array $settings) : void
     {
         $this->settings = $settings;
-
-        return $this;
     }
 
     /**
      * Gets the data (in the preferred format)
      *
      * @return \stdClass
      */
-    public function getData()
+    public function getData() : ?\stdClass
```

Comment:
I agree, I don't think it should be `null`able, but there is currently test(s) that pass back `null` from mocks of these. Created #201 to look into this further, but I'll leave as is for this PR.